docs/public/content/admin/tutorials/registry.md

```diff
@@ -0,0 +1,49 @@
+# Push Docker images on a registry
+
+!!! warning "Official @@PROJECT_NAME@@ images"
+    @@PROJECT_NAME@@ does **not** officially publish any docker image on any public registry such as Docker hub or Quay.io. That said it provides a Bash script to push local images to any Docker registry.
+
+## Pre-requisites
+
+- git
+- bash
+- docker
+
+## Generate @@PROJECT_NAME@@ images
+
+In order to generate the Docker images to upload on the registry you should generate @@PROJECT_NAME@@'s JAVA archives. This can be done following the first part of the [Deploy on Docker tutorial](../docker#build-docker-images). To briefly summarize you should build and run an image which will populate your local directories with the @@PROJECT_NAME@@ JARs and WARs. In the end of this preliminary step you should have your images locally, this can be tested with the command `docker images | grep vulnerability-assessment-tool`.
+
+```sh
+# sample output for `docker images` command
+vulnerability-assessment-tool-patch-lib-analyzer @@PROJECT_VERSION@@ fbe5ec6de811  22 hours ago 103MB
+vulnerability-assessment-tool-rest-backend       @@PROJECT_VERSION@@ 277217bc35b2  22 hours ago 136MB
+vulnerability-assessment-tool-rest-lib-utils     @@PROJECT_VERSION@@ 53bbb929895d  22 hours ago 127MB
+vulnerability-assessment-tool-frontend-bugs      @@PROJECT_VERSION@@ fab5925fe785  22 hours ago 316MB
+vulnerability-assessment-tool-frontend-apps      @@PROJECT_VERSION@@ 191ce235c420  22 hours ago 317MB
+vulnerability-assessment-tool-haproxy            @@PROJECT_VERSION@@ 37948fae374e  22 hours ago 20.9MB
+vulnerability-assessment-tool-postgresql         @@PROJECT_VERSION@@ c7a17e4f4cda  22 hours ago 70.8MB
+```
+
+## Push the images to a registry
+
+A script was created to push the images to a remote Docker registry. [This script](https://github.com/SAP/vulnerability-assessment-tool/blob/master/docker/push-images.sh) simply tags the images and pushes them towards the registry.
```

Comment:
A script was created to push the images to a local Docker registry running inside your organization. [This script](https://github.com/SAP/vulnerability-assessment-tool/blob/master/docker/push-images.sh) simply tags the images and pushes them towards the registry.